### PR TITLE
Dependences version

### PR DESCRIPTION
**Pull request by kelson42**
Depends on https://github.com/openzim/libzim/pull/656

### CODE REVIEW

**Comment by kelson42** [comment]
@veloman-yunkan This is ready to review, basically the same construt like in openzim/libzim.

File: src/version.cpp [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ * MA 02110-1301, USA.
+ */
+
+#include <iostream>
+#include <sstream>
+
+#include <version.h>
+#include <zim/zim.h>
+#include <kiwix_config.h>
+#include <unicode/uvernum.h>
```
Comment (veloman-yunkan) [inline src/version.cpp:26]: Didn't this header cause problems under Windows?

**Review by veloman-yunkan — COMMENTED** [review]
Also the commit message is inaccurate

File: src/version.cpp (hunk 1 above)
Comment (kelson42) [reply src/version.cpp:26]: @veloman-yunkan No, the problem with libzim was the libicu should not be included if libzim is not compiled with libxapian (which is the case for Windows CI). That said,  `unicode/uversion.h` seems slightly better `unicode/uvernum.h` but without any impact of any sort (both work fine).

**Comment by kelson42** [comment]
@veloman-yunkan Yes, have put a new message to the commit. Looks good now?

File: include/version.h [hunk 2]
```diff
@@ -27,6 +27,7 @@ namespace kiwix
 {
   typedef std::vector<std::pair<std::string, std::string>> LibVersions;
   LibVersions getVersions();
+  std::string getVersionsString();
```
Comment (veloman-yunkan) [inline include/version.h:30]: What about slightly enhancing the `printVersions()` function instead?

```
void printVersions(std::ostream& out = std::cout);
```

Then `getVersionsString()` can be replaced with

```
std::ostringstream oss;
zim::printVersions(oss);
kiwix::printVersions(oss);
std::string versionsString = oss.str();
```

Comment (kelson42) [reply include/version.h]: Fixes in last commit

**Review by veloman-yunkan — APPROVED** [review]
Please don't forget to clean up the commit history before merging

**Pull request merged**